### ISSUE TITLE
Pagerank 

### ISSUE THREAD

**Issue by danzhechen** [mikeizbicki/cmc-csci145-math166#155]
Hi there, 

I am going through the PageRank paper right now. And here a thing that I do not understand. 
In this equation 5.1, how can we reach the last equation. It seems that the article does not include a direct formula to translate P to P bar, we need to make every row non-zero, but what is a in this case? 

<img width="736" alt="Screenshot 2024-09-10 at 3 09 50 PM" src="https://github.com/user-attachments/assets/5b140f98-4b3b-4fd4-ab59-e85de30a6b7d">

Thank you for your help in advance.

**Comment by mikeizbicki** [comment]
You're right that the paper doesn't do a good job with defining $\bar P$.  It's first defined in words in Section 3.1 (and that's the definition I used in class/have in my quiz key notes), but then it is re-defined in an un-numbered formula in the very last sentence of Section 4.  The formula is $\bar P = P + \mathbf a \mathbf v^T$, and they define the meaning of $\mathbf a$ in the preceding sentence.  I remember also being frustrated by the lack of a straightforward "definition location" when I first read the paper many years ago.
